build/yaml/templates/dotnet-build-test-steps.yml

```diff
@@ -8,7 +8,12 @@ steps:
   displayName: 'Use .Net Core sdk 3.1.x'
   inputs:
     version: 3.1.x
-    
+
```

Comment:
should we remove the task before this the 3.1.x?

Reply:
No, there is one multitarget test project, so we need this task.